x/valset-pref/msg_server_test.go

```diff
@@ -339,6 +355,15 @@ func (suite *KeeperTestSuite) TestUnDelegateFromValidatorSet() {
 			setExistingDelegations: true,
 			expectPass:             true,
 		},
+		{
+			name:           "Undelegate extreme amounts",
```

Comment:
Is this with the intention of indicating extreme amount in terms of truncation? Or the amount itself?

Reply:
yup that is to check if there is any truncation issue

Reply:
sweet, noticed that you have clarified the test case name, thanks!